clouddriver-elasticsearch/src/main/java/com/netflix/spinnaker/clouddriver/elasticsearch/model/ElasticSearchEntityTagsProvider.java

```diff
@@ -461,6 +468,14 @@ private static EntityTags prepareForWrite(ObjectMapper objectMapper, EntityTags
 
     copyOfEntityTags.getTags().forEach(entityTag -> entityTag.setValue(entityTag.getValueForWrite(objectMapper)));
 
+    String application = copyOfEntityTags.getEntityRef().getApplication();
```

Comment:
crux of the change here ... this is called anytime we index something in elastic search